v2/README.md

```diff
@@ -1,3 +1,19 @@
-## Make it So
+<img width="1062" alt="Make it So Version 2" src="https://github.com/user-attachments/assets/57b9cdef-3fa7-4620-a33e-32db9c704ff4" />
 
-Make it So version 2 is coming soon.
\ No newline at end of file
+## Make it So Version 2
+
+This is a sample Android app that demonstrates how to use Firebase Authentication, Crashlytics and Cloud Firestore. Make it So is a simple to-do list application that allows the user to add and edit to-do items, add flags and priorities, and mark the tasks as completed.
+
+## Setting up
+
+In order for this app to work, you will need to create a [Firebase project](https://firebase.google.com/):
+
+* Clone this repository
+* Create a Firebase project in the [Firebase Console](https://console.firebase.google.com/)
+* Follow [these steps](https://firebase.google.com/docs/android/setup#register-app) to register Make it So app in your Firebase project
+* Follow [these steps](https://firebase.google.com/docs/android/setup#add-config-file) to add the Firebase Android configuration file to Make it So
+* [Create a Cloud Firestore database](https://firebase.google.com/docs/firestore/quickstart#create) in your Firebase project
+* [Enable Anonymous Authentication](https://firebase.google.com/docs/auth/android/anonymous-auth#before-you-begin) in your Firebase project
+* [Enable Email/Password Authentication](https://firebase.google.com/docs/auth/android/password-auth#before_you_begin) in your Firebase project
+* Run the app using Android Studio Ladybug+ on a device/emulator with API level 23 or above
+* Create your first to-do item in the app
```

Comment:
Maybe we should use an ordered list instead of unordered

Reply:
Done!